process/block/poolsCleaner/miniBlocksPoolsCleaner.go

```diff
@@ -82,8 +82,13 @@ func (mbpc *miniBlocksPoolsCleaner) cleanMiniblocksPools(ctx context.Context) {
 		case <-time.After(sleepTime):
 		}
 
+		startTime := time.Now()
 		numMiniblocksInMap := mbpc.cleanMiniblocksPoolsIfNeeded()
-		log.Debug("miniBlocksPoolsCleaner.cleanMiniblocksPools", "num miniblocks in map", numMiniblocksInMap)
+		elapsedTime := time.Since(startTime)
+
+		log.Debug("miniBlocksPoolsCleaner.cleanMiniblocksPools",
```

Comment:
why the same log 2 times ?

Reply:
Its not duplicated. The below one display the number of mbs cleaned if they were, and tthis one display the number of mbs which are in the whole map, and could be cleaned in the future if they will remain unprocessed. Also the elapsed time displayed below refers to the time used to remove the mbs from cacher, and this one refers to the whole time consumed by this method -> cleanMiniblocksPoolsIfNeeded

Reply:
Its not the same. The below one display the number of mbs cleaned if they were, and tthis one display the number of mbs which are in the whole map, and could be cleaned in the future if they will remain unprocessed. Also the elapsed time displayed below refers to the time used to remove the mbs from cacher, and this one refers to the whole time consumed by this method -> cleanMiniblocksPoolsIfNeeded